superset-frontend/src/types/react-table-config.d.ts

```diff
@@ -64,8 +64,10 @@ import {
   UseSortByOptions,
   UseSortByState,
 } from 'react-table';
+import { ColumnSizer } from 'react-virtualized';
 
 declare module 'react-table' {
+  type columnSize = 'xs' | 'sm' | 'md' | 'lg' | 'xl';
```

Comment:
This probably should be `ColumnSize`, especially since that's how it's referenced on line 134